src/main/resources/templates/admin.html

```diff
@@ -89,6 +89,36 @@ <h4 class="header-panel" style="text-align: center">New Rating</h4>
                 <button class="arrow-button" type="submit">Create</button>
             </form>
         </div>
+        <div class="col-sm">
+            <h4 class="header-panel" style="text-align: center">New User</h4>
+            <br>
+            <form action="#" th:action="@{/admin/user/create}" th:object="${userDto}" method="post">
+                <div class="form-group">
+                    <label for="firstName"> First Name : </label>
+                    <input id="firstName" th:field="*{firstName}" />
+                </div>
+                <div class="form-group">
+                    <label for="lastName"> Last Name : </label>
+                    <input id="lastName" th:field="*{lastName}" />
+                </div>
+                <div class="form-group">
+                    <label for="userName"> User name (nick) * : </label>
+                    <input id="userName" th:field="*{userName}" />
+                    <p class="error" th:if="${#fields.hasErrors('userName')}" th:errors="*{userName}" style="color: red">UserName error.</p>
+                </div>
+                <div class="form-group">
+                    <label for="password"> Password * : </label>
+                    <input id="password" th:field="*{password}" />
+                    <p class="error" th:if="${#fields.hasErrors('password')}" th:errors="*{password}" style="color: red">Password error.</p>
+                </div>
+                <div class="form-group">
+                    <label for="birthDate"> Password * : </label>
```

Comment:
`<label for="birthDate"> Password * : </label>` ... copy paste mistake: birthDay~password

Reply:
right, thanks